app/src/helpers/index.ts

```diff
@@ -9,18 +9,31 @@ export const chainConfigs: Readonly<
     endpoint: 'https://api.etherscan.io',
     apiKey: process.env.ETHERSCAN_API_KEY,
   },
-  goerli: {
+  'goerli-ethereum': {
```

Comment:
now there's another file that has to be adjusted along with these here:

https://github.com/valiafetisov/superhack/blob/778c92ea6dde8318cf2fff56be6d084bf291bcb3/app/src/helpers/getProvider.ts#L2

i suggest: 

  1. Create type in `types.ts` to specify the keys that are allowed here.
  2. adjust the linked above file along with the current file to include the endpoint for the goerli optimism

list of networks that ethers.js knows by key string is available at "Supported Networks" section on the site https://docs.ethers.org/v5/api/providers/api-providers/#EtherscanProvider